website/meta/sidebars.json

```diff
@@ -133,6 +134,7 @@
                     { "text": "Corpus", "url": "/api/corpus" },
                     { "text": "InMemoryLookupKB", "url": "/api/inmemorylookupkb" },
                     { "text": "KnowledgeBase", "url": "/api/kb" },
+                    { "text": "Large Language Models", "url": "/api/large-language-models" },
```

Comment:
I would put them under "Pipeline" alphabetically because `llm` is a pipeline component just like `textcat`.